interpreter.js

```diff
@@ -1945,6 +1945,23 @@ Interpreter.prototype.nativeToPseudo = function(nativeObj) {
       nativeObj instanceof RegExp) {
     return this.createPrimitive(nativeObj);
   }
+
+  if (nativeObj instanceof Function) {
+    var wrapper = (function(interpreter, nativeObj){
+      return function() {
+        return interpreter.nativeToPseudo(
+          nativeObj.apply(interpreter,
+            Array.prototype.slice.call(arguments)
+              .map(function(i) {
+                return interpreter.pseudoToNative(i);
+              })
+          )
+        );
+      };
+    })(this, nativeObj);
```

Comment:
I obviously didn't understand the purpose of the `wrapper` function at the time. Thanks for pointing out.